pants.travis-ci.toml

```diff
@@ -36,6 +36,10 @@ timeout_maximum = 590
 # NB: See `test.pytest`.
 timeout_maximum = 590
 
+[pytest]
+# NB: See `test.pytest`.
+timeout_maximum = 590
```

Comment:
I'm still not certain that we want this. Also NB that with remote execution this does not result in 590 because of the queue buffer time we add. (Or did you remove that recently?)

Reply:
> Also NB that with remote execution this does not result in 590 because of the queue buffer time we add. (Or did you remove that recently?)

The server side now applies timeouts, so they should be much closer to accurate. The queue buffer time only comes into play now if the server fails to apply the timeout.

Reply:
Awesome. Still, though, because we can run tests in parallel with v2, I don't think it makes as much sense to eagerly fail at 590 seconds.

Reply:
Maybe. I think that this change is orthogonal to removing the max though. This change is still a draft because it will likely involve adding lots of per-target timeouts.

Reply:
I ended up removing the max since pantsd was hitting it... oy. Thanks!